canonical/canonical.com
# move Jira config file to one level higher to make it work

## Done

Apparently a Jira config file (that allows creation of Jira tickets from GH issues) has to be put under `.github` folder instead of `workflows`.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- This change can be test after it's merged to `main` branch. It doesn't break anything, so it's safe to merge.
